### PR TITLE
adds Grove back

### PR DESCRIPTION
**Pull request by sandikbarr**

### CODE REVIEW

File: stacks/container-apps/root.yml [hunk 1]
```diff
@@ -573,47 +573,47 @@ Resources:
...
+          Fn::ImportValue:
+            !Sub "${InfrastructureNotificationsStackName}-OpsDebugMessagesSnsTopicArn"
+        OpsErrorMessagesSnsTopicArn:
+          Fn::ImportValue:
+            !Sub "${InfrastructureNotificationsStackName}-OpsErrorMessagesSnsTopicArn"
+        EnvironmentType: !Ref EnvironmentType
+        EnvironmentTypeAbbreviation: !Ref EnvironmentTypeAbbreviation
+        EcrRegion: !Ref EcrRegion
+        SecretsBase: !Ref SecretsBase
+        ContainerMemory: !Ref ContainerMemory
```
Comment (cavis) [inline stacks/container-apps/root.yml]: Note to self: at some point, we should start using less memory for frontend apps.  Maybe passing through `FrontendCpu` and `FrontendMemory` from the `stacks/root.yml`, and using those here.  But that's a separate issue.

File: stacks/container-apps/root.yml [hunk 2]
```diff
@@ -573,47 +573,47 @@ Resources:
...
+        ContainerMemoryReservation: !Ref ContainerMemoryReservation
+        ContainerCpu: !Ref ContainerCpu
+        PlatformALBListenerPriorityPrefix: !Ref GrovePlatformALBListenerPriorityPrefix
+        EcrImageTag: !Ref GroveEcrImageTag
+        SecretsVersion: !Ref GroveSecretsVersion
+        AppName: "grove"
+        CreateWorker: "false"
+        ContainerPort: "4204"
+        HealthCheckPath: "/api/v1"
```
Comment (cavis) [inline stacks/container-apps/root.yml]: Does this path really exist for this app?  May just want `"/"` (or anything that returns a 200).

File: stacks/container-apps/root.yml (hunk 1 above)
Comment (cavis) [reply stacks/container-apps/root.yml]: Opened #334

File: stacks/container-apps/root.yml (hunk 2 above)
Comment (sandikbarr) [reply stacks/container-apps/root.yml]: nope, not sure what jingle is doing with it. _so many details_

**Pull request merged**